mediagrains/gsf.py

```diff
@@ -1902,7 +1902,8 @@ def __init__(self,
     def __enter__(self) -> OpenGSFEncoder:
         if not isinstance(self.file, RawIOBase) and not isinstance(self.file, BufferedIOBase):
             raise ValueError("To use in synchronous mode the file must be a synchronously writeable file")
-        self._open_encoder = OpenGSFEncoder(self.file,
+        file = cast(IO[bytes], self.file)
+        self._open_encoder = OpenGSFEncoder(file,
```

Comment:
Very very minor: I would do this inline rather then assign to a variable as `cast` is just providing type hints and an assignment might appear to be more than that. However, it's fine as is.